multicluster/config/overlays/leader-global/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+namespace: changeme
```

Comment:
could leave a comment above describing what name change is expected

Reply:
None of these are actually needed for leader-global, so I removed them.